BrowserKit/Sources/WebEngine/WKWebview/WKWebViewProvider.swift

```diff
@@ -32,8 +32,10 @@ struct DefaultWKWebViewProvider: WKWebViewProvider {
             return nil
         }
 
-        // TODO: FXIOS-7898 #17643 Handle WebView a11y label
+        // TODO: FXIOS-7898 #17643 Handle WebView a11y label and identifier
```

Comment:
Curious why needs to be commented?

Reply:
I could uncomment `webView.accessibilityIdentifier = "contentView"` but I think we should use the a11y identifiers from our [identifiers file](https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Application/AccessibilityIdentifiers.swift). It's a bit more work to pass it down to the WebEngine, but since we already need to pass `.WebViewAccessibilityLabel`, might as well do it right